pss/pss_test.go

```diff
@@ -70,6 +71,117 @@ func TestTrojanChunkRetrieval(t *testing.T) {
 
 }
 
+func TestPssMonitor(t *testing.T) {
```

Comment:
there is a lot of duplicated code from the `TestTrojanChunkRetrieval` func here, is there a need for a separate test? could we not just extend the one we have?

if not, let's extract

Reply:
They are similar, personally I prefer to test the code separately and then join them if necessary.
This will be merged.

Reply:
thanks, will resolve when this is taken care of

Reply:
Please describes what the test does in a comment

Reply:
I will add a comment to this test, the idea was to merge both test, but I think the best thing is keep them separated and extract stuff in common.

Reply:
I was extracting the code in both test and I did not like how I was resolving it.

I think maybe extracting common code in both test `TestPssMonitor` and `TestTrojanChunkRetrieval` is a bit overhead in this stage.

We will have more test in this test suite, I think we should wait for extracting the common code.